Tests/SonarScanner.MSBuild.TFS.Tests/BuildVNextCoverageReportProcessorTests.cs

```diff
@@ -145,8 +145,7 @@ public void ProcessCoverageReports_VsCoverageXmlPathProvided_NotCoverageXmlFileA
 
             // Assert
             result.Should().BeTrue();
-            converter.AssertConvertNotCalled();
-            testLogger.AssertWarningsLogged(0);
+            converter.AssertExpectedNumberOfConversions(1);
```

Comment:
We want to test that the pre-existing behaviour has been preserved i.e. the user has provided a value for the property, we still try the conversion, but don't update the property value.
It would be better to set the config setting to a known value in line 139 above, and assert that it hasn't changed.

Reply:
I've added an assertion